socorro/cron/crontabber_app.py

```diff
@@ -99,6 +98,25 @@ def jobs_converter(path_or_jobs):
 
 
 class CronTabberApp(CronTabberBase, App):
+    config_defaults = {
+        'always_ignore_mismatches': True,
+
+        'crontabber': {
+            'database_class': 'socorro.external.postgresql.connection_context.ConnectionContext',
+
```

Comment:
I think this one got lost:

```
'job_state_db_class': {
    'database_class': 'socorro.external.postgresql.connection_context.ConnectionContext'
}
```

Either that or you changed the way it gets set by setting `resource.postgresql.database_class` below.

If it's the latter, then `resource.postgresql.database_class` value will cover both `crontabber.database_class` as well as `crontabber.job_state_db_class.database_class` and thus we don't need to set `database_class` above.